src/EpicRoadmap/react/Components/EpicRoadmapView.tsx

```diff
@@ -54,6 +54,14 @@ class EpicRoadmapViewContent extends React.Component<IEpicRoadmapViewProps, IEpi
             return (
                 <Spinner size={SpinnerSize.large} className="loading-indicator" label="Loading..." />
             );
+        } else if (uiState === UIStatus.Error) {
+            return (
+                <MessageBar
+                    messageBarType={MessageBarType.error}
+                    isMultiline={false}
+                >{"Epic Roadmap is not supported for Basic process."}
```

Comment:
Is this basic process the only thing that could cause a UIStatus.Error? Seems like we'd want to choose a more specific type like the other UIStatus checks are doing below. Something like UIStatus.BasicProcess.

Reply:
We should not depend if the process is Basic or not ... we just need to detect if there are less than 4 levels of backlogs (e.g. Taskbacklog, Requirement Backlog and atleast two portfolio backlogs).

Reply:
Agreed.